driver/network_driver.go

```diff
@@ -286,12 +286,12 @@ func (d NetworkDriver) CreateEndpoint(request *network.CreateEndpointRequest) (*
 		// the request if the profile already exists.
 		profile := &api.Profile{
 			Metadata: api.ProfileMetadata{
-				Name: networkData.Name,
-				Tags: []string{networkData.Name},
+				Name:   networkData.Name,
+				Labels: map[string]string{"projectcalico.org/group": networkData.Name},
```

Comment:
Had a quick chat with @neiljerram about this, and seems like using what we used for tags as the key for the label with `""` value should work fine, even with the upgrade